### PR TITLE
[enocean] Migrate to bnd build

### PR DESCRIPTION
**Pull request by Hilbrand**
Automigrated with https://gist.github.com/Hilbrand/a7127d0fb7afd36124a78a248453ff03

### CODE REVIEW

**Comment by kaikreuzer** [comment]
@Hilbrand Could you please rebase once more?

**Comment by Hilbrand** [comment]
I'm going to fix the conflicts once #5055 is handled.

**Comment by davidgraeff** [comment]
Other PR is merged

File: addons/pom.xml [hunk 1]
```diff
@@ -17,7 +17,6 @@
   <name>openHAB Add-ons</name>
 
   <modules>
-    <module>binding</module>
```
Comment (Hilbrand) [inline addons/pom.xml]: :tada:

**Pull request merged**